lisa/features/security_profile.py

```diff
@@ -117,12 +117,17 @@ def enabled(self) -> bool:
 
 SecureBootEnabled = partial(
     SecurityProfileSettings,
-    security_profile=search_space.SetSpace(
-        True, [SecurityProfileType.SecureBoot, SecurityProfileType.CVM]
-    ),
+    security_profile=search_space.SetSpace(True, [SecurityProfileType.SecureBoot]),
```

Comment:
If change this, it looks no case flexible for both scenarios.

Reply:
At this moment, I don't think any of the secure boot test cases (TPM) are passing on CVM.

Reply:
It's used by `verify_secureboot_compatibility`. If this test case is for secureboot only, it's ok to me.

Reply:
This test case has never passed on CVM. This could be a test bug or a limitation of CVM. The error is
```
AssertionError: [This OS image is not compatible with Measured Boot.
Get unexpected exit code on cmd ['sudo', '/usr/bin/mbinfo']] Expected <[0]> to contain item <1>, but did not.
```